docs/DEVELOPMENT.md

```diff
@@ -37,3 +37,14 @@ Many editors also have plugins for running Rubocop and helping you to stay withi
 * [Atom](https://atom.io/packages/linter-rubocop)
 * [Sublime Text](https://github.com/SublimeLinter/SublimeLinter-rubocop)
 * [VS Code](https://github.com/misogi/vscode-ruby-rubocop)
+
+## Advanced Search with Algolia
```

Comment:
Please add a newline between the header and text content:

* [MD022 - Headers should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines)